cognitedata/reveal
# fix: allow overloading default style with black (no) color

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-596

## Description :pencil:


The node appearance allows default style color and previously-assigned colors to be overriden by [0, 0, 0] (no color).

## How has this been tested? :mag:



Unit test,  running live script in docs.

## Test instructions :information_source:


Copy-paste the code
```
model.setDefaultNodeAppearance({ color: new THREE.Color(1, 0, 0) });
model.assignStyledNodeCollection(nodes, { color: new THREE.Color(0, 0, 0) });
```

at the end of the first live code block in the doc page about "Filtering nodes by Node ID or properties". The life boats at the back of the platform should remain orange.
## Checklist :ballot_box_with_check:
